### PR TITLE
[DRFT-885] Remove unused NodeJS package from base image

### PR DESCRIPTION
**Pull request by johnsonm325**
This PR removes NPM and its is dependencies including NodeJS, which are not used by drift-backend but their presence in the base image provides potential pathway for security vulnerabilities.

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,9 @@ FROM registry.access.redhat.com/ubi8/python-38
 
 USER 0
 RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    yum remove -y npm \
```
Comment (petrblaho) [inline Dockerfile:7]: There are missing `&&` as on lines above and below.

File: Dockerfile [hunk 2]
```diff
@@ -4,7 +4,9 @@ FROM registry.access.redhat.com/ubi8/python-38
...
     yum install -y hostname shared-mime-info && \
+    yum upgrade -y --security \
```
Comment (petrblaho) [inline Dockerfile:9]: Ditto for missing `&&`.

**Review by petrblaho — CHANGES_REQUESTED** [review]
Build process fails as it tries to run one big mangled command.
Add `&&` as noted and it should work.

**Review by petrblaho — APPROVED** [review]
LGTM!

**Comment by sthirugn** [comment]
> Build process fails as it tries to run one big mangled command. Add `&&` as noted and it should work.

Great catch, Petr thanks

**Pull request merged**